apl-utils/src/main/java/com/apollocurrency/aplwallet/apl/util/cert/CertificateHolder.java

```diff
@@ -43,13 +43,19 @@ public int size() {
     }
 
     public void readCertDirectory(String path, boolean loadPrivateKey) {
-        CertificateLoader cl = new CertificateLoader();
         File dir = new File(path);
         if (dir.exists() && dir.isDirectory()) {
             File[] filesList = dir.listFiles();
             for (File f : filesList) {
                 if (f.isFile() && f.canRead()) {
-                    ApolloCertificate vc = cl.loadCertificate(f.getAbsolutePath());
+                    ApolloCertificate vc=null;
+                    try {
+                        vc = ApolloCertificate.loadPEMFromPath(f.getAbsolutePath());
```

Comment:
Underneath InputStream might not be closed internally. Check is twice pls.

Reply:
fixed